content/10.blackbox.md

```diff
@@ -1,17 +1,15 @@
+In ML, a model's interpretability refers to the ability to identify the discriminative features used for learning and that influence or sway the predictions.
+ML models vary widely in terms of interpretability, ranging from being fully transparent to being a “black-box”.
+In the former case, a clear description of the features' importance can found, such as with logistic regression or decision trees. 
+However, DL models are harder to interpret compared to other ML models because of their large number of parameters and the non-linear relationships between their features.
 
-There are, however, many strategies to interpret a DL models.
-For example, autoencoders (AE) is a family of unsupervised methods that aim to learn a new (encoded) representation and minimize the error between the new representation and the input data. 
-Tan et al., [@doi:10.1142/9789814644730_0014] used a denoising AE to summarize key features from breast cancer dataset.
-The authors could map encoded features to clinical characteristics relevant to the disease. 
+Model interpretability is notably important in the biomedical field, where the decisions of models can mean the difference between life and death.
+Many authors attribute the lack of pervasiveness of DL tools in healthcare because of the inability to understand what these models learn [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
+Luckily, there are many strategies to interpret both ML and DL models.
+In fact, due to the increasing demand for interpretable models, the literature on the topic is currently growing at an exponential rate [@arxiv:2001.02522].
+Therefore, instead of recommending specific methods for either DL-specific or general-purpose model interpretation, we suggest consulting a recent overview.
+We particularly recommend [@url:https://christophm.github.io/interpretable-ml-book/] which is freely available and continually updated.
 
-Model transparency is notably important in the biomedical field.
-Many authors attribute the lack of pervasiveness of deep learning tools in healthcare because of the inability to understand what these models learn [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
-In conclusion, we encourage beginners of the DL to train in parallel a base model that is easier to interpret. 
-In case the difference in accuracy is too high to trade-off with the DL model, pre-training AE may help to dissect which are discriminative features. 
-Alternatively, algorithms based on Garson's work [@tag:garson-interpreting] can help to investigate the weights of a DL model to better understand it [**TODO detail Garson’s algorithm?**].
+In conclusion, analyze your DL models to identify how their inputs are interacting as they generate their predictions, especially when your models are being used for critical decision making.
```

Comment:
This is not really addressed in this section anymore. Before it was suggested to train both simple and complex models, which is not a bad suggestion in many cases.

Reply:
Sorry, can you clarify your comment a little bit? Are you suggesting adding back our suggestion to train a parallel simple model for the purposes of interpretation (in addition to baselining)?

Reply:
This sentence says that the conclusion is to analyze deep learning models to identify how their inputs are interacting. Our paragraph above mostly says these models are considered among the most difficult to interpret but new techniques are emerging.

I don't know if we want to change our conclusion to be about the paragraph that's here, or if we want to revise our guidelines to provide advice/examples around why it's so important to analyze and interpret models (which would probably require discussion of things like https://medium.com/@jrzech/what-are-radiological-deep-learning-models-actually-learning-f97a546c5b98 from @jrzech